src/main/kotlin/com/example/fhirvalidator/shared/PrePopulatedValidationSupport.java

```diff
@@ -0,0 +1,255 @@
+package com.example.fhirvalidator.shared;
```

Comment:
same as before

Reply:
The intent here is to alter the HAPI FHIR libraries and remove our copy. 
This is for exploratory work with the terminology server.

Reply:
Same applies to the HybridTerminologyServiceValidationSupport class. This  ideally will go in HAPI FHIR.